### ISSUE TITLE
Added tasks are not quoted

### ISSUE THREAD

**Issue by olafhering** [sgaraud/gnome-extension-taskwarrior#13]
If the task contains a ' adding will fail because the string is not quoted according to gnome-shell:

`
org.gnome.Shell.desktop[1986]: GLib.ShellError: Text endete, bevor ein passendes Anführungszeichen für ' gefunden wurde. (Der Text war »task add Xen's qdisk«)
`

Creating the same task with tasknc works.

**Comment by sgaraud** [comment]
Correct. Currrently there is no escaping done behind the scene. As explained in [official doc](https://taskwarrior.org/docs/escapes.html) you will need to escape some special character or quote some text very much like when using _task_ on the command line. I understand it is not 
Thanks to point towards tasknc, I will look into what was done for escaping inputs.
I will add escaping as a feature in the next release.
